### PR TITLE
Add missing dependency for control_msgs

### PR DESCRIPTION
**Pull request by stefanscherzinger**
As per subject

### CODE REVIEW

**Comment by fmauch** [comment]
:-D https://github.com/UniversalRobots/Universal_Robots_ROS_passthrough_controllers/commit/53b4bc84b04166eb81cdca29695927b6d113af8b

File: package.xml [hunk 1]
```diff
@@ -78,6 +80,7 @@
   <exec_depend>actionlib</exec_depend>
   <exec_depend>cartesian_control_msgs</exec_depend>
   <exec_depend>cartesian_interface</exec_depend>
+  <exec_depend>control_msgs</exec_depend>
```
Comment (fmauch) [inline package.xml:83]: Should it be an exec depend?

**Review by fmauch — COMMENTED** [review]
I would rather use #4 as it adds more missing dependencies.

**Comment by stefanscherzinger** [comment]
Yes, I'll close this in favor of #4.

**Pull request closed without merging**